microk8s-resources/actions/enable.host-ip.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+read -ra IP_ADDRESS <<< "$1"
```

Comment:
Could you tweak this a bit so the arguments passed are named ie `host-ip:ip=10.0.1.1` vs `host-ip:10.0.1.1`. You can look at the registry addon on how this can be done. Thanks

Reply:
sure, I wanted to avoid the repeat of word 'ip'. Another suggestion would be `host-ip:address=10.0.1.1`. So, `host-ip:ip` or `host-ip:address`?

Reply:
Perhaps call it `microk8s.host-access`?

Reply:
@ktsakalozos you call :)

Reply:
done as proposed initially by @ktsakalozos

Reply:
Lets do `host-address:ip=x.y.z.j` to keep @balchua happy :)

Reply:
done